src/import/kv_importer.rs

```diff
@@ -178,6 +178,14 @@ impl KVImporter {
             }
         }
     }
+
+    /// Get importer's version
+    /// version string formats like 'v3.0.0-123abc'
+    pub fn get_version(&self) -> Result<String> {
+        let pkg_version = env!("CARGO_PKG_VERSION");
+        let commit_num = &env!("GIT_HASH")[..6];
+        Ok(format!("v{}-{}", pkg_version, commit_num))
```

Comment:
It is probably better to return two strings separately. You may need to update the gRPC to return two strings.

```proto
message GetVersionResponse {
    string version = 1;    // "3.0.2" or "4.0.0-alpha" etc, no "v" prefix
    string commit = 2;     // the 40-digit commit hash
}
```